viper-toolchain-versions.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+SILVER_REF="f645539a04e76d70337d7b5b108e7f9a9de9bc7a"
```

Comment:
This corresponds to [this commit](https://github.com/viperproject/silver/commit/f645539a04e76d70337d7b5b108e7f9a9de9bc7a). We can no longer use the Viper release as `AbstractGobraTests` needs to specialize what a comment is and what not such that the annotation matching correctly works for `.go` test cases